modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -591,6 +591,21 @@ void ONNXImporter::populateNet(Net dstNet)
             }
             layerParams.set("num_output", layerParams.blobs[0].size[1] * layerParams.get<int>("group", 1));
             layerParams.set("bias_term", node_proto.input_size() == 3);
+
+            const DictValue* outShape = layerParams.ptr("output_shape");
+            if (outShape && outShape->size() == 4)
```

Comment:
* Please check if there is `"output_shape"` parameter by `layerParams.has("output_shape");`
* It's more convenient to use `DictValue outShape = layerParams.get("output_shape");`
* If `outShape.size() != 4` it's better to throw a `StsNotImplemented` exception because we don't know how to manage it.

Reply:
Done.